### DISCUSSION TITLE
New Contributor: Where to Start?

### DISCUSSION THREAD

**Discussion by Rajz7** [JdeRobot/jderobot.github.io#283] [General]
Hi developers I'm a second year student learning pytorch I wish to contribute to your organization in ml / ai space it will be really helpful if you guide me what to learn specifically or suggest me some good first issues
thank you

**Comment by Salma-1011** [comment]
HI I'm CSE student i have completed DSA and learning python I wish to contribute to your organization in open 3D Engine as robotics simulator if you guide me

**Reply by jmplaza** [reply]
Hi @Salma-1011 , sure,

 just install and play with O3DE on your machine, follow the [application instructions for GSoC-2025 with JdeRobot](https://jderobot.github.io/activities/gsoc/2025#application-instructions-for-gsoc-2025). Playing with RoboticsAcademy is also a good step. Once you solved the programming tests and fill the form with your data, please contact with the mentors for feedback about your proposal of that Project#6. 

Cheers,

**Reply by Salma-1011** [reply]
Ok

**Comment by jmplaza** [comment]
Sure @Rajz7 , 

feel free to contact the potential mentors of [project#4](https://jderobot.github.io/activities/gsoc/2025#project-4-robotics-academy-new-exercise-on-end-to-end-visual-control-of-an-autonomous-vehicle-using-deeplearning) and [project#10](https://jderobot.github.io/activities/gsoc/2025#project-10-extend-detectionmetrics-gui-ci-workflow-and-object-detection). They are the more related to ML/AI for the [ideas list from JdeRobot for GSoC2025](https://jderobot.github.io/activities/gsoc/2025).

Beyond GSoC-2025, we have a RoboticsClub and internships where we you can learn and become a long term contributor on ML/AI topics in JdeRobot software.
